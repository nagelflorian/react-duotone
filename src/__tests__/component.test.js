/* global describe, it, expect */
import React from 'react';
import renderer from 'react-test-renderer';
import DuotoneImageComponent from '../component';

describe('DuotoneImageComponent', () => {
  it('renders the image correctly', () => {
    const tree = renderer.create(
      <DuotoneImageComponent
        src="https://avatars.githubusercontent.com/u/7649376?v=3"
        primaryColor="#66FA75"
        secondaryColor="#1904C7"
      />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('renders additional attributes correctly', () => {
    const tree = renderer.create(
      <DuotoneImageComponent
        src="https://avatars.githubusercontent.com/u/7649376?v=3"
        primaryColor="#66FA75"
        secondaryColor="#1904C7"
        alt="GitHub Avatar"
        height="56"
        width="56"
      />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
