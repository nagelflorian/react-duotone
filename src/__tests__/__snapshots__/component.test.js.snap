// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DuotoneImageComponent renders additional attributes correctly 1`] = `
<img
  alt="GitHub Avatar"
  height="56"
  src=""
  width="56"
/>
`;

exports[`DuotoneImageComponent renders the image correctly 1`] = `
<img
  src=""
/>
`;
